{{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/base_templates/layout.pt

```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html metal:define-macro="layout">
-<html lang="{{ '{{request.locale_name}}' }}">
+<html lang="${ '{{request.locale_name}}' }">
```

Comment:
I think you got the syntax inverted. Cookiecutter uses Jinja2 on the outer element to render the inner element.
```suggestion
<html lang="{{ '${request.locale_name}' }}">
```